tests/test_ote_api.py

```diff
@@ -11,7 +11,6 @@
 from typing import Optional
 
 import numpy as np
-import pytest
```

Comment:
@LeonidBeynenson is it Ok to remove this one?

Reply:
pytest module is not used in test_ote_api.

Reply:
Yes, at the moment pytest is not used. Forgotten import.